### ISSUE TITLE
Add API function/events for modifying signal post-calculation

### ISSUE THREAD

**Issue by Sniperhid** [IDI-Systems/acre2#205]
**Description:**
- This would be really useful to allow other mods/missions to interact with the signal model but adding an event style system (where the return values would influence the signal strength) - This could be used to add boosting/degradation/jamming.

**Comment by jonpas** [comment]
Mostly addressed by #1087, which lets you do things in the following way:

```sqf
// Register custom signal function
[{
    // ACRE signal processing
    private _coreSignal = _this call acre_sys_signal_fnc_getSignalCore;
    _coreSignal params ["_Px", "_maxSignal"];

    // Modify signal
    // ...

    // return final values
    [_Px, _maxSignal]
}] call acre_api_fnc_setCustomSignalFunc;
```

Not event-style, but pretty straight-forward. @Sniperhid

**Comment by jonpas** [comment]
Addressed in #1087.
